src/main/java/com/gdschongik/gdsc/domain/email/api/OnboardingUnivEmailController.java

```diff
@@ -35,10 +33,10 @@ public ResponseEntity<Void> sendUnivEmailVerificationLink(
     }
 
     @Operation(summary = "학교 인증 메일 인증하기", description = "학교 인증 메일을 인증합니다.")
-    @GetMapping("/verify-email")
+    @PatchMapping("/verify-email")
     public ResponseEntity<Void> sendUnivEmailVerificationLink(
-            @RequestParam(VERIFY_EMAIL_REQUEST_PARAMETER_KEY) String verificationCode) {
-        univEmailVerificationService.verifyMemberUnivEmail(verificationCode);
+            @RequestBody @Valid UnivEmailTokenVerificationRequest verificationToken) {
```

Comment:
dto는 request라는 이름으로 받고 있습니다~

```suggestion
            @RequestBody @Valid UnivEmailTokenVerificationRequest request) {
```

Reply:
메일로 보내는 링크는 `/onboarding/verify-email?%token={token value}`의 형태던데,
그걸 클릭하면 인증되는 방식 아닌가요?
만약 맞다면 파라미터가 아닌 body로 토큰을 받는 이유가 궁금합니다!

Reply:
> 메일로 보내는 링크는 `/onboarding/verify-email?%token={token value}`의 형태던데, 그걸 클릭하면 인증되는 방식 아닌가요? 만약 맞다면 파라미터가 아닌 body로 토큰을 받는 이유가 궁금합니다!

이거 하나만 설명 부탁드립니다!